client/my-sites/checkout/src/components/wp-order-review-line-items.tsx

```diff
@@ -130,6 +131,10 @@ export function WPOrderReviewLineItems( {
 		);
 	}, [ responseCart.products ] );
 
+	const hasWPCOMPlanInCart = useMemo( () => {
```

Comment:
Minor, but I don't think this needs to be memoized. Boolean values passed as dependencies to `useCallback()` are compared by value not by reference, so even if this is recreated on each render, it won't accidentally trigger a recreation of the callback unless it changes.

Also, the overhead of calling `some` on the cart items on every render is relatively small, so I'm not sure this provides a lot there.

It's also fine if you want to keep it.